src/usdb_syncer/meta_tags/deserializer.py

```diff
@@ -156,6 +156,45 @@ def __init__(self, video_tag: str, logger: Log) -> None:
                 case _:
                     logger.warning(f"unknown key for meta tag: '{pair}'")
 
+    def __str__(self) -> str:
```

Comment:
There is already a serializer for meta tags which we use in the meta tags dialog. However, I have a local branch that refactors that code, so we should probably merge that in first. Only I cannot push it until next weekend, unfortunately.

Reply:
Great, then let’s wait until that branch is merged and then we continue from there.